### DISCUSSION TITLE
Visual Studio 2019 compile errors on windows system

### DISCUSSION THREAD

**Discussion by dwyanexu419** [OpenRadioss/OpenRadioss#696] [Q&A] [answered]
Hello, I have installed Visual studio 2019 and intel oneAPI 2022 in my windows system. The environment of openRadioss on windows is built with cygwin, and the make package and perl package are installed in cygwin. The Visual studio solution can be built successfully, however, when the solution is opened for build with vs2019, an MSB8066 error is reported with exit code 9009. I don't know how to fix this error？
thank you!

**Comment by dwyanexu419** [comment]
The build log says: 'perl' is not an internal or external command, nor is it a runnable program. But I have installed perl package in cygwin

**Comment by OlivierWienholtz** [comment, marked as answer]
Hello, 

An issue was made of this discussion, Pls have a look at the link bellow : 

https://github.com/OpenRadioss/OpenRadioss/issues/695

Cheers,
Olivier
